conftest.py

```diff
@@ -15,7 +15,6 @@
 
 from reporting import pytest_metadata, pytest_json_modifyreport, add_extra_json_metadata # noqa
 
-settings.register_profile("xp_default", deadline=800)
```

Comment:
What is the deadline without this here?

Reply:
200 good catch, Ill make it default to 800 again!

Reply:
We might need to think about deadlines a bit more (but in a separate issue). Deadline errors are very common, even with 800. It's not just with JAX, I see them with every library. I disabled the deadline in the array-api-compat and array-api-strict CI. It looks like [numpy does too](https://github.com/numpy/numpy/blob/e1bf1d635016a0118f7d77bf8931071a2c74ad20/.github/workflows/linux.yml#L251). I think there's just a fundamental incongruity with the hypothesis deadline and the `arrays` strategy and the test suite.

Reply:
> I think there's just a fundamental incongruity with the hypothesis deadline and the arrays strategy and the test suite.

Yep thats a sound observation. For now I've made it default to 800 like before, and we can mull it over.